test/inductor/test_codecache.py

```diff
@@ -132,12 +135,6 @@ def fn(x, y):
         self.assertEqual(counters["inductor"]["fxgraph_cache_miss"], 1)
         self.assertEqual(counters["inductor"]["fxgraph_cache_hit"], 1)
 
-        # But we expect different code if the tensors are aliased.
```

Comment:
This was never correct and was passing by mistake. Enabling the FakeTensor cache exposes the mistake (by mistake ;) ).